nominal/core/asset.py

```diff
@@ -86,20 +88,27 @@ def nominal_url(self) -> str:
         # TODO (drake): move logic into _from_conjure() factory function to accomodate different URL schemes
         return f"https://app.gov.nominal.io/assets/{self.rid}"
 
-    def add_dataset(self, data_scope_name: str, dataset: Dataset | str) -> None:
+    def add_dataset(
+        self,
+        data_scope_name: str,
+        dataset: Dataset | str,
+        *,
+        series_tags: Mapping[str, str] | None = None,
+        offset: timedelta | IntegralNanosecondsDuration | None = None,
+    ) -> None:
         """Add a dataset to this asset.
 
         Assets map "data_scope_name" (their name within the asset) to a Dataset (or dataset rid). The same type of
```

Comment:
I notice that `ref_name` is used in runs, but `data_scope_name` in assets. They both seem to be a mapping of 'name within X -> rid`. If these are fundamentally different concepts, perhaps something to be explained in the user guide. Otherwise, should the name be consistent?

Reply:
Yeah you're right, this is a product decision that we need to make!